lib/shared/tlv/include/tlv/TlvBer.hxx

```diff
@@ -244,6 +244,7 @@ public:
         if ((*dataIt & BitmaskTagFirstByte) != TagValueLongField) {
             tag.push_back(*dataIt);
             tagNumber = *dataIt & BitmaskTagShort;
+            bytesParsed++;
```

Comment:
I wondered how the tests didn't pick this up, but I see now the tests are all using two-byte (long/extended) tags. So let's update the tests to include 1-byte tags as well, which might expose additional issues.